libraries/helpers.rb

```diff
@@ -30,6 +30,37 @@ def keys_from_url(url)
       end
     end
 
+    # Determine of a shell exists on disk
+    #
+    # @return [Boolean]
+    def shell_exists(shell_path)
```

Comment:
Idiomatically you should add a `?` to the end of this method name so `shell_exists?`.